docs/tutorials/setup_acquisition/storage.md

```diff
@@ -30,9 +30,12 @@ The output of that script will be:
 <DeviceIdentifier Storage "Zarr">
 <DeviceIdentifier Storage "ZarrBlosc1ZstdByteShuffle">
 <DeviceIdentifier Storage "ZarrBlosc1Lz4ByteShuffle">
+<DeviceIdentifier Storage "ZarrV3">
+<DeviceIdentifier Storage "ZarrV3Blosc1ZstdByteShuffle">
+<DeviceIdentifier Storage "ZarrV3Blosc1Lz4ByteShuffle">
 ```
 
-`Acquire` supports streaming data to [bigtiff](http://bigtiff.org/) and [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html).
+`Acquire` supports streaming data to [bigtiff](http://bigtiff.org/), [Zarr V2](https://zarr-specs.readthedocs.io/en/latest/v2/v2.0.html), and [Zarr V3](https://zarr-specs.readthedocs.io/en/latest/specs.html).
```

Comment:
You might mention that we support [OME](https://ngff.openmicroscopy.org/latest/)-Zarr specifically, for both Zarr V2 and Zarr V3.